includes/templates/story-templates/travel-vertical.html

```diff
@@ -1,5 +1,5 @@
 <!-- wp:amp/amp-story-page {"mediaType":"image","focalPoint":{"x":0.6161616161616161,"y":1},"autoAdvanceAfterDuration":0} -->
-<amp-story-page style="background-color:#ffffff" id="cd109143-b365-4e99-9c44-f5bd707cf9c1" class="wp-block-amp-amp-story-page"><amp-story-grid-layer template="fill"><amp-img layout="fill" src="AMP_STORY_TEMPLATES_URL/quote-bg.jpg" style="object-position:61.61616161616161% 100%"></amp-img></amp-story-grid-layer><amp-story-grid-layer template="fill"></amp-story-grid-layer>
+<amp-story-page style="background-color:#ffffff" id="cd109143-b365-4e99-9c44-f5bd707cf9c1" class="wp-block-amp-amp-story-page"><amp-story-grid-layer template="fill"><img layout="fill" src="AMP_STORY_TEMPLATES_URL/quote-bg.jpg" object-position="61.61616161616161% 100%"/></amp-story-grid-layer><amp-story-grid-layer template="fill"></amp-story-grid-layer>
```

Comment:
Template version needs to be bumped

Reply:
Looks like the template version bump gets lost with merging :D

Changing.